augur/tasks/init/celery_app.py

```diff
@@ -101,27 +102,43 @@ def augur_handle_task_failure(self,exc,task_id,repo_git,logger_name,collection_h
                 session.commit()
 
     def on_failure(self,exc,task_id,args, kwargs, einfo):
-        repo_git = args[0]
+        repo_git = self._extract_repo_git(args, kwargs)
         # log traceback to error file
         self.augur_handle_task_failure(exc, task_id, repo_git, "core_task_failure")
 
+    def _extract_repo_git(self, args, kwargs):
```

Comment:
Would it maybe be a better option to standardize the calling convention rather than dynamically locating the argument post-call? Or, is there some reason why enforcing call-side standardization is not possible?